model/src/w3fldsmd.F90

```diff
@@ -359,13 +360,13 @@ SUBROUTINE W3FLDO ( INXOUT, IDFLD, NDS, NDST, NDSE, NX, NY,     &
 #ifdef W3_DEBUGFLS
        WRITE(740+IAPROC,*) '1 : W3FLDSMD FNAME=', FNAME(:I)
 #endif
-              OPEN (NDS,FILE=FPRE//FNAME(:I),FORM=FORM,ERR=803,       &
+              OPEN (NDS,FILE=FPRE//FNAME(:I),FORM=FORM,convert=file_endian,ERR=803,       &
```

Comment:
I just removed this line + two other additions in model/src/w3fldsmd.F90 and tested regtests with both little and big endians. It seems the fix from the finished jobs. I'll let you know once all tests are done, so you can revert them.